etest/__init__.py

```diff
@@ -59,6 +60,9 @@ def etest(dry_run, fast, jobs, quiet, verbose, ebuilds):
     output_lock = threading.Lock()
     jobs_limit_sem = threading.BoundedSemaphore(value=jobs)
 
+    if arch != "amd64":
```

Comment:
This should probably check if the system arch matches or we can make things easier and unconditionally use qemu.  If you do want to make it condition, it looks like platform holds the answer: https://docs.python.org/3/library/platform.html#platform.machine

Reply:
The QEMU interpreter containers dont run under anything that isnt amd64, so until we figure out a way to let the user disable that feature entirely we can let that as-is.